webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -498,16 +550,16 @@ def _define_style_ensemble(index: int, x_axis: pd.Index) -> Dict:
     return style
 
 
-def _update_corr_index_dict(
+def _get_first_observation_index(
     response_x_axis: Union[pd.Index, list], observation_x_axis: pd.DataFrame
 ) -> int:
     x_axis_default_observation = _get_first_observation_x(observation_x_axis)
     if isinstance(response_x_axis, pd.Index):
-        updated_index = response_x_axis.get_loc(x_axis_default_observation)
+        calculated_index = response_x_axis.get_loc(x_axis_default_observation)
```

Comment:
Sorry for late comment, but would not make more sense to be first_observation_index instead of calculated_index?

Reply:
yes indeed, good catch : ) gonna change and squash all into one commit!

Reply:
done!